configs/selfsup/byol/r50_bs4096_ep300_syncbn_grad_accmulate.py

```diff
@@ -79,16 +78,16 @@
         pipeline2=train_pipeline2))
 # additional hooks
 custom_hooks = [
-    dict(type='BYOLHook', end_momentum=1.)
+    dict(type='BYOLHook', end_momentum=1., update_interval=16)
```

Comment:
You'd better merge the two `update_interval` since they bind to each other. In this way, when some users want to change the update_interval, it will be changed both in two positions.
E.g.,
`update_interval=16 # graduate accumulation iterations for BYOL`
`custom_hooks = [
    dict(type='BYOLHook', end_momentum=1., update_interval=update_interval)]`
`optimizer_config = dict(update_interval=update_interval, use_fp16=use_fp16)`